### PR TITLE
Install mlnx-ofa_kernel-devel on Red Hat family OS

### PR DESCRIPTION
**Pull request by ahmam**
modify the package install to 	mlnx-ofa_kernel-devel

### CODE REVIEW

File: tasks/install-RedHat.yml [hunk 1]
```diff
@@ -2,5 +2,5 @@
 - name: Install MLNX_OFED packages for Red Hat OS family
   ansible.builtin.package:
     name:
-      - kmod-mlnx-ofa_kernel
+      - 	mlnx-ofa_kernel-devel
```
Comment (btravouillon) [inline tasks/install-RedHat.yml:5]: You can't remove `kmod-mlnx-ofa_kernel`. Please install both packages.

**Review by btravouillon — CHANGES_REQUESTED** [review]

File: tasks/install-RedHat.yml (hunk 1 above)
Comment (ahmam) [reply tasks/install-RedHat.yml]: oui j'ai oublié de le remettre  fixed

**Review by achillemyette1 — DISMISSED** [review]

File: tasks/install-RedHat.yml [hunk 2]
```diff
@@ -2,5 +2,6 @@
 - name: Install MLNX_OFED packages for Red Hat OS family
   ansible.builtin.package:
     name:
-      - kmod-mlnx-ofa_kernel
```
[the remark this answers is not in the document]
Comment (ahmam) [reply tasks/install-RedHat.yml:5]: fixed

[the remark this answers is not in the document]
Comment (btravouillon) [reply tasks/install-RedHat.yml:5]: There is no need to remove `kmod-mlnx-ofa_kernel` from this list.

**Review by btravouillon — CHANGES_REQUESTED** [review]

**Review by btravouillon — APPROVED** [review]
Looks good to me. 👍

**Review by pbouchez — APPROVED** [review]

**Pull request merged**